src/app/page.tsx

```diff
@@ -1,12 +1,13 @@
 "use client";
 
-
+import { Animated, Basic, bounce, Combined } from '../styles/styles'
```

Comment:
Are you using Combined, Basic ??? if not, remove it.  Also remove from the `style.js` file

Reply:
removed extra variables